language/move-prover/tests/sources/regression/data_invariant_for_mut_ref_arg.exp

```diff
@@ -0,0 +1,10 @@
+Move prover returns: exiting with verification errors
+error: unknown assertion failed
```

Comment:
I am wondering whether we could generate more concrete error message, e.g. the assertion fails because `s` might not be in a consistent state?

Reply:
This is going to be hard... This falls in the line of explaining why an assertion fail (e.g., due to a missing assumption), which is theoretically intractable.